ALS.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 16 16:07:50 2022
+
+@author: Kirsteenng
+"""
+
+import os
+import time
+
+# spark imports
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import UserDefinedFunction, explode, desc
+from pyspark.sql.types import StringType, ArrayType
+from pyspark.mllib.recommendation import ALS
+
+# data science imports
+import math
+import numpy as np
+import pandas as pd
+
+# visualization imports
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+%matplotlib inline
+
+
+# spark config
+spark = SparkSession \
+    .builder \
+    .appName("movie recommendation") \
+    .config("spark.driver.maxResultSize", "96g") \
+    .config("spark.driver.memory", "96g") \
+    .config("spark.executor.memory", "8g") \
+    .config("spark.master", "local[12]") \
+    .getOrCreate()
+# get spark context
+sc = spark.sparkContext # Get or instantiate a SparkContext and register it as a singleton object.
+ 
+# path config
+data_path = '/Users/Kirsteenng_1/Data Science/Movie Recommendation system'
```

Comment:
Can you update your README to include exact steps for which dataset you downloaded?  You might also want to rethink the data_path.  I might want to replicate this by reusing your code.  I need steps on getting setup so I can simply execute your code without any issues :)